ldi-extensions/ldes-discoverer/README.md

```diff
@@ -0,0 +1,97 @@
+# Linked Data Interactions LDES Discoverer
```

Comment:
Not a fan of copy-pasting our docs in the readmes.. but depends on how team feels about this